xml/System.Security.Cryptography/AesCng.xml

```diff
@@ -631,7 +631,8 @@
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
-## Remarks  
+## Remarks 
+ For AES, the legal key sizes are 128, 192, and 256 bits.
  If you've created the <xref:System.Security.Cryptography.AesCng> object using an existing persisted key and you set the value of the <xref:System.Security.Cryptography.AesCng.KeySize%2A> property,  the persisted key will no longer be used and an ephemeral key will be used instead. If you need to use the persisted key again, a new instance of <xref:System.Security.Cryptography.AesCng> class needs to be created.  
```

Comment:
There should be a blank line between these statements so that they become separate paragraphs on the docs site.

Reply:
i'm not sure if I put newline in the right place?

Reply:
Yep, looks like what I'd expect.